### DISCUSSION TITLE
Go Version 1.17 and greater, along with v1.22.7+k3s1 suggestion

### DISCUSSION THREAD

**Discussion by james-millner** [carlosedp/cluster-monitoring#152] [Ideas]
Hi Carlos! 

I've used your repository a few times thank you very much! Its absolutely brilliant! 

I've come back to it on a WSL setup I'm currently running on. I have some Raspberry Pi's setup with K3s.io and your repo really helped me get up and running. However I ran into two small notes that I thought I'd try and raise a PR for, however I would prefer to discuss it first as I'm sure you've probably had a note on it. 

I currently tried running this with `go_version_go1.17.8` and as a result go wasn't happy with the Makefile's use of go get. I was able to change this to:

`go install` 

Which allowed the installation to continue 👍 

Secondly, I also tried to run on a cluster this using` v1.22.7+k3s1` which worked great, but some of the generated resources made use of the K8s resource:

`Warning: spec.template.spec.nodeSelector[beta.kubernetes.io/os]: deprecated since v1.14; use "kubernetes.io/os" instead`

Those were basically the two changes I needed to make, to then have a happy K3s cluster monitoring setup! Thank you once again!

Just wanted to sign this off with that I really appreciate the work you've done on this setup. I've shared this space a few times with some colleagues of mine, its a brilliant. Thank you again!!

![image](https://user-images.githubusercontent.com/15033304/158085519-aa8b047a-d155-43a9-aa53-d7659c50bc61.png)

Woohoo!

**Comment by robmit68** [comment]
Hello James which files you need to change the warning below

Warning: spec.template.spec.nodeSelector[beta.kubernetes.io/os]: deprecated since v1.14; use "kubernetes.io/os" instead

Thank you
